src/transformState.js

```diff
@@ -5,7 +5,30 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const count in actions) {
+    switch (actions[count].type) {
+      case 'addProperties':
+        for (const key in actions[count].extraData) {
+          state[key] = actions[count].extraData[key];
```

Comment:
I guess you can add new properties with Object.asign()

Reply:
Got what you meant, but now I'm wondering what the difference between these approaches in general and in this task. Will ask on q&a :)